src/ducks/balance/BalanceWithBanksJobs.jsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+import { useBanksContext } from 'ducks/context/BanksContext'
+import Balance from 'ducks/balance/Balance'
+
+const BalanceWithBanksJobs = props => {
```

Comment:
Are you sure that it takes props? We should avoid spreading `...props` when we know there is no props

Reply:
No it doesn't. I removed it